test/FastTests/Voron/Trees/FreeSpaceTest.cs

```diff
@@ -466,8 +466,7 @@ public void CanUpdateMaxConsecutiveRanges()
 
             using (var tx = Env.WriteTransaction())
             {
-                Env.FreeSpaceHandling.FreePage(tx.LowLevelTransaction, 4);
-
+                Env.FreeSpaceHandling.FreePage(tx.LowLevelTransaction, 5);
```

Comment:
Can you please elaborate on this test adjustment change?

Reply:
my understanding of this line in the test is that we want to check the rollback due to the comment below
> // not commiting on purpose, this should simulate a rollback and clear the in memory state

Since we already freed page 4 previously, freeing it again would cause a double free exception now, so I just freed a different page and ensure that the assertion further in the test will pass

Reply:
Makes sense. Thanks